apps/webClient/src/components/BulbIcon/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+interface Props {
+  height?: number;
+  width?: number;
+}
+
+const BulbIcon: React.FC<Props> = ({ height = 70, width }) => {
+  const svgProps = {};
+  if (height) {
+    Object.assign(svgProps, { height });
+  }
+  if (width) {
+    Object.assign(svgProps, { width });
+  }
+  return (
+    <svg
+      id="prefix__Layer_1"
```

Comment:
not a huge deal, but I would prefer if we removed the `id` attributes from out svgs